src/MusicAnnouncementsConfig.cs

```diff
@@ -59,16 +65,16 @@ private void AddTitle()
 			});
 		}
 
-		private void AddCheckbox()
+		private void AddCheckbox(Configurable<bool> optionText)
```

Comment:
This should have a `float y` parameter added like `AddDivider()`, otherwise both checkboxes will be placed in the same spot.
```suggestion
		private void AddCheckbox(Configurable<bool> optionText, float y)
```
**Before:**
![image](https://user-images.githubusercontent.com/57483089/219381584-1c487a00-14d2-4317-96ad-4575644b70f1.png)
**After:**
![image](https://user-images.githubusercontent.com/57483089/219382244-10b8e5e6-30da-4114-80e3-b11bfbec4f13.png)